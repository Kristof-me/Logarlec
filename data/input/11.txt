load map.data
create student s1 r1
create transistor t1 s1
create transistor t2 s1
use s1 t1
drop s1 t1
move s1 d12
status s1 location
use s1 t2
status s1 location