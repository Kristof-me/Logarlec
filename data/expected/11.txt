r2
r1
